statuspage/src/components/CanaryModal.vue

```diff
@@ -0,0 +1,73 @@
+<!-- This component encapsulates a modal that is displayed -->
+<!-- when clicking on canary                               -->
+<template>
+  <!--  Component modal doc: : https://bootstrap-vue.org/docs/components/modal-->
```

Comment:
I would add a comment at the `v-b-modal` declaration too, even if more terse, as that would be the first encounter. 

This is good though, so leave it in. Just maybe add another pointer.

Reply:
Added a comment over there as well :)